File: Javascript/InsertionSort.js
function insertionSort (array) {
    for (let itemCounter = 0; itemCounter < array.length; itemCounter += 1) {
        let item = array[itemCounter]
        let locationCounter = 0
        while (item > array[locationCounter]) locationCounter += 1
        for (let counter = itemCounter; counter > locationCounter; counter -= 1) array[counter] = array[counter - 1]
        array[locationCounter] = item
    }
    return array
}

File: Python/InsertionSort.py
def insertionSort(array):
    for itemCounter in range(len(array)):
        item = array[itemCounter]
        locationCounter = 0
        while item > array[locationCounter]:
            locationCounter += 1

        for i in reversed(range(locationCounter, itemCounter + 1)):
            array[i] = array[i - 1]

        array[locationCounter] = item

    return array
